src/TaylorBot.Net/Core/src/TaylorBot.Net.Core/Embed/EmbedBuilderExtensions.cs

```diff
@@ -11,7 +11,7 @@ public static EmbedBuilder WithUserAsAuthor(this EmbedBuilder embedBuilder, IUse
             if (user.IsBot)
                 author += " 🤖";
 
-            var avatarUrl = user.GetAvatarUrlOrDefault();
+            var avatarUrl = user.GetGuildAvatarUrlOrDefault();
```

Comment:
This is a good change! Let's use it in the Last.fm commands as well that set the Last.fm username as author. I would leave the global one for UserInfo though